helpdesk_ticket_multiple_attachments_from_mobile/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Ticket Attachment Enhancer From Mobile",
+    "summary": """
+        Enable multiple fields for ticket creation via mobile to allow uploading multiple photos taken from the device camera
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
```

Comment:
We're inheriting from an AGPL-3 licensed addon, that is more restricting than LGPL-3. Then, this addon should have the same license than base addon